frontend/src/pages/RunList.tsx

```diff
@@ -17,7 +17,7 @@
 import * as React from 'react';
 import CustomTable, { Column, Row, CustomRendererProps } from '../components/CustomTable';
 import Metric from '../components/Metric';
-import RunUtils, { MetricMetadata } from '../../src/lib/RunUtils';
+import RunUtils, { MetricMetadata, ExperimentInfo } from '../../src/lib/RunUtils';
```

Comment:
It seems you forgot to delete this? I didn't find any usage.

Reply:
I moved the declaration of ExperimentInfo from RunList.tsx to RunUtils.ts. ExperimentInfo is still used on [line 50](https://github.com/kubeflow/pipelines/pull/1824/files#diff-4121e43f42fc68ebceee87b4789159d5L50) in RunList.tsx.

Reply:
Got it, missed existing usage.